docs/reference/modules/node.asciidoc

```diff
@@ -109,6 +109,12 @@ node.ingest: false <3>
 <2> Disable the `node.data` role (enabled by default).
 <3> Disable the `node.ingest` role (enabled by default).
 
+ifdef::include-xpack[]
+NOTE: These settings apply only when {xpack} is not installed. To create a
+standalone master-eligible node when {xpack} is installed, see <<modules-node-xpack,{xpack} node settings>>.
```

Comment:
I would say "dedicated" instead of "standalone" here, that's consistent with the other sections and consistent with what is already on the page regarding dedicated master-eligible nodes.